src/main.cc

```diff
@@ -763,7 +763,6 @@ main(int argc, char **argv)
 		 */
 		struct fiber_attr attr;
 		fiber_attr_create(&attr);
-		attr.flags &= ~FIBER_IS_CANCELLABLE;
```

Comment:
This means that from now on waking up the shutdown fiber will stop the event loop, thus crashing Tarantool:

```
tarantool> require('fiber').find(102):wakeup()
---
...

tarantool> tarantool: /home/vlad/src/tarantool/tarantool/src/lib/core/fiber.c:390: fiber_call_impl: Assertion `callee->f != NULL && callee->fid != 0' failed.
Aborted
```

This is unacceptable. Please make sure the `on_shutdown` fiber is safe against spurious wakups and add a test case (in a separate commit).

Also, please squash the two commits, because they are two sides of the same coin. We don't need two doc tickets, either.

Reply:
Is not it related to #7434?

Reply:
> Is not it related to #7434?

I don't think so. I guess this happens, because when the tarantool exits normally, it stops the event loop before running shutdown triggers, making yields impossible. Maybe a dup of https://github.com/tarantool/tarantool/issues/6801.

Reply:
Fixed.